File: Android/Martinshare/app/src/main/java/mobile/martinshare/com/martinshare/API/protocols/GetVersionHistoryProtocol.java
package mobile.martinshare.com.martinshare.API.protocols;

import mobile.martinshare.com.martinshare.API.POJO.EintraegeList;

/**
 * Created by Modestas Valauskas on 24.11.2015.
 */
public interface GetVersionHistoryProtocol {
    public void startedGetting();
    public void notLoggedIn();
    public void noInternet();
    public void unknownError(String text);
    public void gotVersionHistory(EintraegeList eintraege);
}

File: iOS/Martinshare/Martinshare/GetVersionHistoryProtocol.swift
//
//  GetVersionHistoryProtocol.swift
//  Martinshare
//
//  Created by Modestas Valauskas on 24.11.15.
//  Copyright © 2015 ModestasV Studios. All rights reserved.
//

import Foundation

protocol GetVersionHistoryProtocol {
    
    func startedGetting()
    func notLoggedIn()
    func unknownError(_ string: String)
    func noInternet()
    func got(_ eintraege: Array<EintragDataContainer>)
    
}
